images/win/scripts/Installers/Update-AndroidSDK.ps1

```diff
@@ -75,12 +75,34 @@ Install-AndroidSDKPackages -AndroidSDKManagerPath $sdkManager `
                           -AndroidSDKRootPath $sdkRoot `
                           -AndroidPackages $androidToolset.additional_tools
 
+# NDKs
+$ndkLTSMajorVersion = $androidToolset.ndk.lts
+$ndkLatestMajorVersion = $androidToolset.ndk.latest
+
+$ndkLTSFullVersion = Get-AndroidPackagesByName -AndroidPackages $androidPackages `
+                -PrefixPackageName "ndk;" `
+                | Where-Object { $_ -Match "ndk;$ndkLTSMajorVersion.*" } `
+                | Sort-Object -Unique `
+                | Select-Object -Last 1
+
+$ndkLatestFullVersion = Get-AndroidPackagesByName -AndroidPackages $androidPackages `
+                -PrefixPackageName "ndk;" `
+                | Where-Object { $_ -Match "ndk;$ndkLatestMajorVersion.*" } `
+                | Sort-Object -Unique `
+                | Select-Object -Last 1
+
+$androidNDKs = @($ndkLTSFullVersion, $ndkLatestFullVersion)
+
+Install-AndroidSDKPackages -AndroidSDKManagerPath $sdkManager `
+                          -AndroidSDKRootPath $sdkRoot `
+                          -AndroidPackages $androidNDKs
+
 # Android NDK root path.
 $ndkRoot = "$sdkRoot\ndk-bundle"
 # This changes were added due to incompatibility with android ndk-bundle (ndk;22.0.7026061).
 # Link issue virtual-environments: https://github.com/actions/virtual-environments/issues/2481
 # Link issue xamarin-android: https://github.com/xamarin/xamarin-android/issues/5526
-New-Item -Path $ndkRoot -ItemType SymbolicLink -Value "$sdkRoot\ndk\21.3.6528147"
+New-Item -Path $ndkRoot -ItemType SymbolicLink -Value "$sdkRoot\ndk\$($ndkLTSFullVersion.Split(';')[1])"
```

Comment:
Use a separate variable for `$($ndkLTSFullVersion.Split(';')[1]`, because the same expression is used in the line 105&119

Reply:
Moved to separated variables.